### ISSUE TITLE
Add ability to import items via CSV

### ISSUE THREAD

**Issue by mpaddock** [UK-AS-HIVE/Checkouts#1]

**Comment by noahadler** [comment]
Good idea-- one implementation trick I've heard is to make sure to offer a template/example .CSV downloadable from the same UI area you'd do the import.  Otherwise, users will invariably make some small discrepancy that will break the parser, and wonder why it isn't working.
